packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/computeAxisValue.ts

```diff
@@ -200,7 +200,9 @@ export function computeAxisValue<T extends ChartSeriesType>({
     }
 
     const rawTickNumber = getTickNumber({ ...axis, range, domain: axisExtremums });
-    const tickNumber = rawTickNumber / ((zoomRange[1] - zoomRange[0]) / 100);
+    /* If the zoom start and end are the same, `tickNumber` will become infinity, so we should default to 1. */
+    const tickNumber =
```

Comment:
Decided to default to one tick because if zoom range is zero, there is no need to show more than one tick.

Reply:
Should we have a test for this? 🤔

Reply:
Added 👍